AppDashboard/lib/app_dashboard_data.py

```diff
@@ -647,3 +651,65 @@ def can_upload_apps(self):
     except Exception as err:
       logging.exception(err)
       return False
+
+  def get_dash_layout_settings(self, email=None):
+    """ Queries the UserAppServer to see what settings the user has saved
+    for customizing the UI of the Dashboard
+
+    Returns:
+      A JSON string describing the customization layout
+    """
+    default_dash_layout = '''
```

Comment:
my reasoning for making the json string was just proof that I could load from one, the json string will never be constructed in python it will actually be constructed in javascript and sent to the python server to be stored in the datastore